Project 5 For John Beverly

```diff
@@ -59,345 +59,388 @@ Group 2 contains Emanuel, Katie, Josh V (commonly know as V by group member), Ca
 Please note we DID check this against SHACL playground :) 
 enjoy, if possible.
 
-
-1st constraints begins below
-
-@prefix ex: <http://example.org/> .
-
+1.	Constraint for Asymmetry 
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix ex: <http://example.com/> .
 
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 
-@prefix schema: <http://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-ex:RegistrationShape
+ex:AsymmetryShape
 
 a sh:NodeShape ;
 
-sh:targetClass ex:Registration ;
+sh:targetClass ex:relation ;
 
 sh:property [    
 
-sh:path ex:lineRegistration ;    
+sh:path rdf:value ;    
```

Comment:
Interesting, I have never seen rdf:value, what does it do? Also, can we have sh:path without sh:property before it?